lib/src/revset.rs

```diff
@@ -1167,8 +1217,11 @@ fn transform_expression_bottom_up(
                 transform_rec(candidates, f).map(RevsetExpression::Roots)
             }
             RevsetExpression::PublicHeads => None,
-            RevsetExpression::Branches => None,
-            RevsetExpression::RemoteBranches => None,
+            RevsetExpression::Branches(_) => None,
+            RevsetExpression::RemoteBranches {
+                branch_needle: _,
+                remote_needle: _,
+            } => None,
```

Comment:
Nit: could be `RevsetExpression::RemoteBranches { .. }` if you prefer.

Reply:
TIL!